lib/wallaroo/core/data_receiver/data_receiver.pony

```diff
@@ -181,14 +183,25 @@ actor DataReceiver is Producer
       @printf[I32]("Rcvd pipeline msg at DataReceiver\n".cstring())
     end
     if seq_id > _last_id_seen then
-      ifdef "resilience" and debug then
-        Invariant((seq_id - _last_id_seen) == 1)
+      if (seq_id - _last_id_seen) == 1 then
```

Comment:
This assumption only holds if we're in resilience mode. Outside of resilience mode, it's theoretically possible for there to be gaps. That's why this was in an ifdef.